src/reduxStore/slices/networkConfigSlice.ts

```diff
@@ -65,6 +67,12 @@ export const networkConfigSlice = createSlice({
     ) => {
       state.chainID = action.payload;
     },
+    setRoundDuration: (
```

Comment:
I think this should not exist. When we refresh the network, all network props should be set together